scripts/build/build/targets.py

```diff
@@ -237,6 +237,9 @@ def HostTargets():
     for target in targets:
         app_targets.append(target.Extend(
             'all-clusters', app=HostApp.ALL_CLUSTERS))
+        if (HostBoard.NATIVE.PlatformName() == 'darwin'):
+            app_targets.append(target.Extend(
+                'chip-tool-darwin', app=HostApp.CHIP_TOOL_DARWIN))
```

Comment:
in hindsight, having 'chip-tool' and 'chip-tool-darwin' as names is probably very confusing. It is hard to differentiate between `darwin-x64-chip-tool` and `darwin-x64-chip-tool-darwin` at a first glance, especially if you start adding more variants like `darwin-x64-chip-tool-ipv6only`.

How about naming this like `darwin-tool` ? even then it stutters: `darwin-x64-darwin-tool` is bad... maybe `cli-tool` to have `darwin-x64-cli-tool`.

Any other suggestions on naming? I am saying 'chip-tool-darwin' together with 'chip-tool' is bad naming.

Reply:
We have an Issuer for updating the name of the Objective-C (CHIP.framework) implementation. We haven't came up with a decent name.

Reply:
I think chip-objc-framework-tool. This way it specifies what it is using.